tensorrt_llm/_torch/cute_dsl_kernels/blackwell/conv/dense_implicit_gemm_fprop.py

```
@@ -0,0 +1,2034 @@
+            #
+            # Dealloc the tensor memory buffer
+            #
+            tmem.relinquish_alloc_permit()
+            tmem.free(tmem_ptr)
+
+
+@lru_cache(maxsize=1)
+def compile_conv(
+    ncdhw: Tuple[int, int, int, int, int],
+    ktrs: Tuple[int, int, int, int],
+    input: cute.Tensor,
```

Ruff (0.16.1)

[error] Function argument `input` is shadowing a Python builtin

(A002)